src/components/Common/Tooltip.tsx

```diff
@@ -193,7 +193,7 @@ export const TooltipContent = forwardRef<
           context={context.context}
           fill="currentColor"
           strokeWidth={1}
-          className="stroke-gray-400 text-gray-100 dark:stroke-gray-600 dark:text-black"
+          className="stroke-primary text-[var(--bg-layer-0,_#000000)]"
```

Comment:
Why do we use CSS variable here instead of `bg-layer-0`?

Reply:
Actually this thing is needed for arrow in tooltip and as I understand its configured inheriting current color (text color)
This is really exception and I'm not sure that we should allow to use bg-layer-0 in text colors